docker-compose.yml

```diff
@@ -3,31 +3,30 @@ services:
 
   api:
     build:
-      context: api
+      context: perfil
     depends_on:
-      - mongo
+      - db 
     env_file:
       - .env
     ports:
       - "8000:8000"
     volumes:
-      - ./api:/code
+      - ./perfil:/code
 
   notebooks:
     build:
       context: notebooks
-    depends_on:
-      - mongo
     env_file:
       - .env
     ports:
       - "8888:8888"
     volumes:
       - ./notebooks:/code
 
-  mongo:
-    image: mongo:3.6.5-jessie
-    ports:
-      - "27017:27017"
-    volumes:
-      - ./mongo:/data/db
+  db:
+    image: postgres
+    environment:
+      - PGDATA=./perfil/db
```

Comment:
Don't you thinking about putting those variables on `.env` file?

Reply:
No need if we assume people will always use Docker.